VERSIONING_POLICY.md

```diff
@@ -0,0 +1,9 @@
+# Versioning Policy
+
+PureScript can be perceived from two different perspectives:
+1. PureScript-the-application (e.g. using `purs` to compile code)
+1. PureScript-the-library (e.g. building a tool that depends on the [`purescript` package](https://hackage.haskell.org/package/purescript))
+
+This project is versioned using [SemVer 2.0.0](https://semver.org/), not [PVP](https://pvp.haskell.org/) because users of PureScript-the-application are the intended audience. Thus, breaking changes to PureScript-the-application are reflected in this project's version. Since the compiler's artifacts (e.g. the externs format, `CoreFn`) are considered part of PureScript-the-application, a breaking change to these things is reflected in the project version.
```

Comment:
Externs are not a public API, it's only CoreFn.

Reply:
Fixed.